Examples/Examples.playground/Pages/03-UIKitExtensions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,139 @@
+//: [Table of Contents](00-ToC)
+
+//: [Previous](@previous)
+
+import SwifterSwift
+
+//: ## UIKit extensions
+
+//: ### UIAlertController extensions
+
+// Create a new alert controller from string or Error
+let alert = UIAlertController(title: "Couldn't sign in", message: "Invalid username or password!")
+
+// show alert with ease
+alert.show()
+
+//: ### UIButton extensions
+
+let button = UIButton()
+
+// Set title, title color and image for all states at once!
+button.setTitleForAllStates("Login")
+button.setTitleColorForAllStates(UIColor.blue)
+button.setImageForAllStates(UIImage(named: "login")!)
+
+// or set each of them from code or storyboard
+button.titleForHighlighted = "Login"
+
+//: ### UIColor extensions
+
+// Create new UIColor for RGB values
+let color1 = UIColor(red: 121, green: 220, blue: 164)
+
+// Create new UIColor for a hex string (including strings starting with #, 0x or in short css hex format)
+let color2 = UIColor(hexString: "#00F")
+
+// Create new UIColor for a hexadecimal value
+let color3 = UIColor(hex: 0x45C91B)
+
+// Blend two colors with ease
+UIColor.blend(UIColor.red, intensity1: 0.5, with: UIColor.green, intensity2: 0.3)
+
+// Return hexadecimal value string
+UIColor.red.hexString
+
+// Use Google Material design colors with ease
+let indigo = UIColor.Material.indigo
+
+// Use CSS colors with ease:
+let beige = UIColor.CSS.beige
+
+// Return brand colors from more than 30 social brands
+let facebookColor = UIColor.Social.facebook
+
+//: ### UIImage extensions
+
+var image1 = UIImage(named: "logo")!
+
+// Crop images
+let croppedImage = image1.cropped(to: CGRect(x: 0, y: 0, width: 100, height: 100))
+
+// Create UIImage from color
+let image2 = UIImage(color: UIColor.red, size: CGSize(width: 100, height: 100))
+
+// scale to fit width or height
+let scaledImage1 = image2.scaled(toHeight: 50)
+let scaledImage2 = image2.scaled(toWidth: 50)
+
+// Compress images
+let compressedImage = image2.compressed(quality: 0.3)
+
+// get image size
+image2.kilobytesSize
+
+//: ### UIImageView extensions
+
+let imageView = UIImageView()
+
+// Download an image from URL in background
+imageView.download(from: URL(string: "https://camo.githubusercontent.com/5d919daa24d2e934b11f2155f21a0f86fbf3abba/68747470733a2f2f63646e2e7261776769742e636f6d2f5377696674657253776966742f5377696674657253776966742f6d61737465722f4173736574732f6c6f676f2e737667")!, contentMode: .scaleAspectFit, placeholder: image1, completionHandler: nil)
```

Comment:
Interestingly this actually fails. That image is an SVG, which can't be used to create a `UIImage`. Why don't you try the one from the unit tests: https://developer.apple.com/swift/images/swift-og.png ?
Also, you won't actually be able to see the results this way. If you want to let Playgrounds wait for a completion you can start by calling this at the top:
```Swift
import PlaygroundSupport

PlaygroundPage.current.needsIndefiniteExecution = true
```

and then change this `download` call to:
```Swift
imageView.download(from: URL(string: "https://developer.apple.com/swift/images/swift-og.png")!,
                   contentMode: .scaleAspectFit,
                   placeholder: image1,
                   completionHandler: { downloadedImage in
                    downloadedImage
                    PlaygroundPage.current.needsIndefiniteExecution = false
})
```